src/main/java/com/google/cloud/tools/app/api/Configuration.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.app.api;
+
+/**
+ * Base Cloud SDK configuration for configuration spanning multiple commands.
+ */
+public interface Configuration {
+    String getFormat();
```

Comment:
Yea good point.

Regarding your suggestion: the way its designed the cloud sdk implementations only take in an implementation-agnostic configuration. I don't think we want to hard code this config in the action. Unless I misunderstood what you are suggesting?